File: Python Word examples/encrypt/encryptDocument.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.encrypt_api.EncryptApi(configuration)

file_name = "Template.docx"
extra = {
    'new_password': '123',
    'folder': "input",
    'dest_file_path': 'output/EncryptDocument1.docx'}
result1 = api.encrypt_document(file_name, **extra)

File: JavaScript Word Examples/EncryptApiDemo/encryptDocument.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.EncryptApi(apiClient);

    var fileName = "Template.docx";
    var destFilePath = 'output/EncryptDocument.docx';
    var password = "123";
    var opt = { folder: 'input', password: password };
    var response = instance.encryptDocument(fileName, destFilePath, opt, 
        function (error, data, response) {
            if (error) throw error;
        });
})();